styleguide.css

```diff
@@ -276,11 +276,14 @@ button:hover {
     transition: all 0.25s ease-in-out;
 }
 
-form#signin {
+form[name=login_user_form] {
```

Comment:
underscores vs hyphen

Reply:
I think you chose underscores for this one, so I fit your element name